headers/augment_headers.py

```diff
@@ -155,12 +158,14 @@ def _input_header_lines(lines) -> Iterator[Tuple[str, bool, bool]]:
             if not in_comment:
                 in_docstring = False
 
-            yield line, in_comment, in_docstring
+            is_deprecated_macro = not in_comment and line.startswith("DEPRECATED(")
```

Comment:
Let's `lstrip` the line of leading whitespace before checking for `DEPRECATED(`

```suggestion
            is_deprecated_macro = not in_comment and line.lstrip().startswith("DEPRECATED(")
```